docs/user/metrics/downloads.csv

```diff
@@ -8,3 +8,4 @@ Library,PyPI/CRAN Last Week,PyPI/CRAN Last Month,PyPI/CRAN All Time,Conda All Ti
 `TerraTorch`_,318,"1,322","7,037",0,"7,037"
 `SITS`_,120,539,"14,618","78,976","91,743"
 `scikit-eo`_,115,717,"14,700",0,"14,700"
+`otbtf`_,-,-,-,-,-
```

Comment:
Are there any plans to add otbtf to PyPI or Conda Forge so people can more easily install it? Or are there any other ways we can track downloads? I realize it's partly Python, partly C++, so this is more complicated than the average library.

Reply:
We are dependant on the packaging of OTB, which is a C++ library, that is not yet distributed as python package. They are working on this issue, it might be a pip package one day, but not now unfortunately.

The only way we can track OTBTF downloads is the number of docker pull on dockerhub.

Reply:
We can add the number of docker pulls if you want, not sure what granularity they have. Otherwise I'm also fine leaving it blank for now. Thanks for the explanation!